<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentView</class>
 <widget class="QWidget" name="StudentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>408</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="List">
    <widget class="QLabel" name="LS_HeaderLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Books</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="contentWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="shoppingCartButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>370</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>View Shopping Cart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addToCartButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Selected To Cart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewDetailsButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>View Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Details">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>241</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Book Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>311</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>Description...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Cart</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>326</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>By:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>By:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
